images/python/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM scratch
 ARG BUILD_OUTPUT_BASE=build-output
 ARG PYTHON_TAG=v3.11.1
-ARG PYTHON_BINARY=python.wasm
+ARG PYTHON_BINARY=python-wasmedge.wasm
```

Comment:
Was the previous `python.wasm` a different compilation target? It's not clear if the previous `python.wasm` binary was the compiled version for WasmEdge or another target. 

If this is the main one we're using in the container, it's totally fine 😄

Reply:
`python-wasmedge.wasm` is the main one, which we are using for the container. the previous one would never work if this Dockerfile was called by default without explicit arguments